extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/serde/tableExtractors.scala

```diff
@@ -52,10 +53,16 @@ object TableExtractor {
    * @return owner
    */
   def getOwner(v: AnyRef): Option[String] = {
-    // org.apache.spark.sql.connector.catalog.Table
-    val table = invokeAs[AnyRef](v, "table")
-    val properties = invokeAs[JMap[String, String]](table, "properties").asScala
-    properties.get("owner")
+    try {
+      // org.apache.spark.sql.connector.catalog.Table
+      val table = invokeAs[AnyRef](v, "table")
+      val properties = invokeAs[JMap[String, String]](table, "properties").asScala
+      properties.get("owner")
+    } catch {
+      case e: Throwable =>
+        logError("Extract owner from table failed.", e)
```

Comment:
How about logging on debug level? The owner is an optional thing.

Reply:
Done